content/kb/troubleshooting-websocket_handshake_error_with_unexpected_response_code_200.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Troubleshooting: Longhorn-UI: Error during WebSocket handshake: Unexpected response code: 200 #2265"
+author: Clark Hsu
+draft: false
+date: 2021-03-09
+catelogies:
+  - "longhorn-ui"
+---
+
+## Applicable versions
+
+Existing Longhorn versions < v1.1.0 upgrade to Longhorn >= v1.1.0.
+
+## Symptoms
+
+After upgraded Longhorn to version >= v1.1.0, and encounter the following:
+
+Ingress messages:
+
+```
+{"level":"error","msg":"vulcand/oxy/forward/websocket: Error dialing \"10.17.1.246\": websocket: bad handshake with resp: 200 200 OK","time":"2021-03-09T08:29:03Z"}
+```
+
+Longhorn UI messages:
+
+```
+10.46.0.3 - - [19/Feb/2021:11:33:24 +0000] "GET /node/v1/ws/1s/engineimages HTTP/1.1" 200 578 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.150 Safari/537.36 Edg/88.0.705.68"
```

Comment:
What's the difference between this log and correct logs?

Reply:
@cclhsu suggest: next time, please resolve the conversation w/ the reason for the complete context. Thanks.